.github/workflows/release.yml

```diff
@@ -39,6 +47,14 @@ jobs:
           git tag ${{ github.event.inputs.version }} -a -m 'Release ${{ github.event.inputs.version }}'
           git push origin tag ${{ github.event.inputs.version }}
 
+      # Restore the SPMTest app for development purposes
+      - name: Restore SPMTest demo app
+        run: |
+          mv ${{ env.TEMP_DIR }}/SPMTest/ SampleApps/
+          git add SampleApps/SPMTest
+          git commit -m 'Restore SPMTest app for development'
```

Comment:
Similarly, this could be `Post-release: Restory SPMTest app for development`.

Reply:
DONE